models/shopify_graphql.yml

```diff
@@ -0,0 +1,1202 @@
+        description: The unit of measurement for the weight of a single item (e.g., grams, ounces).
+      - name: weight_value
+        description: The numeric weight of a single unit of the line item using the specified `weight_unit`.
+
+  - name: shopify_gql__transactions
+    description: Each record represents a transaction in Shopify.
+    columns:
+      - name: transactions_unique_id
+        description: Unique key representing a transaction. Hashed on 'transaction_id' and 'source_relation'.
+        tests:
+          - unique
+          - not_null
+      - name: transaction_id
+        description: The ID for the transaction.
+      - name: order_id
+        description: The ID for the order that the transaction is associated with.
+      - name: refund_id
+        description: The ID associated with a refund in the refund table.
+      - name: amount_shop
+        description: The amount of money included in the transaction in shop currency.
+      - name: amount_pres
+        description: The amount of money included in the transaction in presentment currency.
+      - name: created_timestamp
+        description: The date and time when the transaction was created.
+      - name: processed_timestamp
+        description: The date and time when a transaction was processed.
+      - name: gateway
+        description: The name of the gateway the transaction was issued through.
+      - name: source_name
+        description: The origin of the transaction.
+      - name: amount_pres_currency_code
+        description: The three-letter code (ISO 4217 format) for the shop currency used for the payment.
+      - name: amount_shop_currency_code
+        description: The three-letter code (ISO 4217 format) for the presentment currency used for the payment.
+      - name: location_id
+        description: The ID of the physical location where the transaction was processed.
+      - name: parent_id
+        description: The ID of an associated transaction.
+      - name: payment_avs_result_code
+        description: The response code from the address verification system.
+      - name: payment_credit_card_bin
+        description: The issuer identification number (IIN), formerly known as bank identification number (BIN) of the customer's credit card.
+      - name: payment_cvv_result_code
+        description: The response code from the credit card company indicating whether the customer entered the card security code, or card verification value, correctly.
+      - name: payment_credit_card_number
+        description: The customer's credit card number, with most of the leading digits redacted.
+      - name: payment_credit_card_company
+        description: The name of the company that issued the customer's credit card.
+      - name: kind
+        description: The transaction's type.
+      - name: receipt
+        description: A transaction receipt attached to the transaction by the gateway.
+      # - name: currency_exchange_id
```

Comment:
Should we remove these?

Reply:
Commenting them back in as these are fields that will be seamlessly brought in when the connector switches to the 2025-07 version of the API